### PR TITLE
Fixed PHP Warning:  curl_errno(): supplied resource is not a valid cURL handle resource

### PR DESCRIPTION
**Pull request by shtse8**

### CODE REVIEW

File: src/Handler/Curl.php [hunk 1]
```diff
@@ -61,10 +61,12 @@ public function send(RequestInterface $request, array $options = array())
...
         $result = curl_exec($ch);
         $headerSize = curl_getinfo($ch, CURLINFO_HEADER_SIZE);
-        curl_close($ch);
+		$errno = curl_errno($ch);
+		$error = curl_error($ch);
+		curl_close($ch);
 
         if ($result === false) {
-            throw new Exception(sprintf('%d - %s', curl_errno($ch), curl_error($ch)));
+            throw new Exception(sprintf('%d - %s', $errno, $error));
```
Comment (ptcong) [inline src/Handler/Curl.php]: Why don't use like this

```
        if ($result === false) {
            $errno = curl_errno($ch);
	    $error = curl_error($ch);
            curl_close($ch);
            throw new Exception(sprintf('%d - %s', $errno, $error));
        }
```
Also every PR, please use PSR2, and correct your indentation.

**Comment by shtse8** [comment]
Because I don't want to put the curl_close() inside the if case. Otherwise, we need to have two lines of curl_close().
If there is no exception is throw, the curl will not be closed.

**Comment by ptcong** [comment]
Ok, correct your indentation before get this PR merged

**Comment by shtse8** [comment]
It's done.

**Comment by ptcong** [comment]
@shtse8 every PR, please create new branch from the master branch, it's better.

**Comment by shtse8** [comment]
I am new in git and don't know too much about branch. How could I create a new branch to create PR?

**Comment by ptcong** [comment]
https://confluence.atlassian.com/bitbucket/branching-a-repository-223217999.html
every time, when you have new feature or bug fixes, clone this project, make new branch and work on it, then use it for new PR

**Comment by shtse8** [comment]
Everytime, I just fork this project and modify it and then create a new PR.
Is the github fork same as clone you mentioned?

**Pull request merged**